packages/fxa-event-broker/config/index.ts

```diff
@@ -27,6 +27,12 @@ const conf = convict({
       doc: 'Refresh interval in seconds for fetching capability updates',
       env: 'CAPABILITY_REFRESH_INTERVAL',
       format: Number
+    },
+    requireCapabilities: {
+      default: true,
+      doc: 'Whether to fail to start if capabiltiies are not available from the auth server',
+      env: 'CAPABILITY_START_ON_FAIL',
```

Comment:
And actually, I think the name of this env var may have contributed to my misreading too, it thought it meant the negation of what it actually is. Would "...FAIL_ON_START" be clearer than "...START_ON_FAIL"?

Reply:
Another one where I wondered the same.